automated/linux/perf/perf.yaml

```diff
@@ -0,0 +1,29 @@
+metadata:
+    name: perf
+    format: "Lava-Test-Shell Test Definition 1.0"
+    description: "Perf is a profiler tool for Linux and is used to collect and analyze performance and trace data.
+                  This test runs following perf commands: record(record events for later reporting), report(break
+                  down events by process, function, etc), stat(obtain event counts) and test(Run sanity tests)."
+    maintainer:
+        - naresh.kamboju@linaro.org
+        - milosz.wasilewski@linaro.org
+    os:
```

Comment:
Add debian and centos?

Reply:
Added debian.
I am thinking if we do not find a way to install perf package on centos. Shall we not include centos here.

Reply:
If it is not needed/tested on centos, then lets don't claim it supports centos. Then we should drop the installation lines for centos as well.